docs/snippets/ov_preprocessing_migration.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from openvino.runtime import Core, Layout, Type
+from openvino.preprocess import PrePostProcessor
```

Comment:
I think this can be replaced with something like:
```suggestion
#! [imports]
from openvino.runtime import Core, Layout, Type
from openvino.preprocess import PrePostProcessor, ColorFormat, ResizeAlgorithm
#! [imports]
```
That way there is no need to repeat same imports in whole code snippet. You may also decide to create aliases for packages and reuse them later. This can be included in general import note in both scenarios, where you tell the user that imports are coming from different packages.
```suggestion
#! [imports]
from openvino.runtime as ov
from openvino.preprocess as preproc # I am not sure about name here
#! [imports]
```